recipes/evolutionary_keras/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "evolutionary_keras" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: 
+  url: https://github.com/N3PDF/evolutionary_keras/archive/v{{version}}.tar.gz
+  sha256: 39dc8ff742cea271b67605c029fb0ae66bbfa1cc2dd5574ee372d0ed81234a83
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
```

This can probably be a `noarch: python`, but add the version constraint below `python >=3.7`

---

I'm doing this because of keras, does it makes more sense to use noarch:python instead and let the keras dependency deal with the constraint on python?

---

```suggestion
  noarch: python
```

I think it is better to follow @scopatz suggestion

---

Oh, now that I saw your other suggestions I understood the msg. Sorry, I'm still new to conda recipes (and thanks!). I'll test noarch and then see whether I truly need to enforce the version.